src/mixins/columns.js

```diff
@@ -57,6 +57,9 @@ export default {
 
     methods: {
         columnsChanged (columns) {
+            if (columns.length === 0) {
+                return;
```

Comment:
Hi @kinow thx for your PR!

First of all, may I ask in what kind of situation do you want to use no columns? But you are right: It's better if it's also supported. 

But I wouldn't do an early return here because their can also be a situation where all the columns are deleted. For example go from 10 to 0 columns.

Reply:
So I would prefer to check if the `columns.length > 0` where we need at least one column and that is on line 72.

Reply:
Hi @arnedesmedt !

Sorry, didn't see your previous comment here.

>First of all, may I ask in what kind of situation do you want to use no columns? But you are right: It's better if it's also supported.

Either dynamically loading the data and the columns, or, in my case, simply creating a prototype where I don't know the columns yet... I was just putting the components in place first before hooking up data from Vuex.

>But I wouldn't do an early return here because their can also be a situation where all the columns are deleted. For example go from 10 to 0 columns.

Oh, got it!

>So I would prefer to check if the columns.length > 0 where we need at least one column and that is on line 72.

Let me see if I can update this PR today. Thanks for the review!

Reply:
commit edited, also rebased my branch onto develop. Let me know if that looks better, or if you would like that it, maybe, had a single combined `if`, instead of two nested.

Reply:
Hai, thx for the changes!